### ISSUE TITLE
可视化

### ISSUE THREAD

**Issue by et123456789** [luow23/INP-Former#24]
作者您好，我现在想将测试图像和热力图进行叠加
def save_img(imgs, anomaly_map, gt, save_root, img_path):
    batch_num = imgs.shape[0]
    for i in range(batch_num):
        img_path_list = img_path[i].split('/')
        class_name, category, idx_name = img_path_list[-4], img_path_list[-2], img_path_list[-1]
        os.makedirs(os.path.join(save_root, class_name, category), exist_ok=True)
        input_frame = denormalize(imgs[i].clone().squeeze(0).cpu().detach().numpy())
        cv2_input = np.array(input_frame, dtype=np.uint8)
        plt.imsave(os.path.join(save_root, class_name, category, fr'{idx_name}_0.png'), cv2_input)
        ano_map = anomaly_map[i].squeeze(0).cpu().detach().numpy()
        ano_map = (ano_map * 255).astype('uint8')
        # 将ano_map转换为热力图
        heatmap = cv2.applyColorMap(ano_map, cv2.COLORMAP_JET)
        # 确保 heatmap 的形状与 cv2_input 一致
        heatmap = cv2.resize(heatmap, (cv2_input.shape[1], cv2_input.shape[0]))

        # 调整heatmap的透明度
        ano_map = cv2.addWeighted(cv2_input, 0.7, heatmap, 0.3, 0)
        plt.imsave(os.path.join(save_root, class_name, category, fr'{idx_name}_1.png'), ano_map)
        gt_map = gt[i].squeeze(0).cpu().detach().numpy()
        plt.imsave(os.path.join(save_root, class_name, category, fr'{idx_name}_2.png'), gt_map, cmap='gray')
        plt.close()

但是最终的结果如图所示

![Image](https://github.com/user-attachments/assets/966fbf0e-30ab-4437-948f-7cc53a256ddd)
之前保存的ano_map是没有问题的

![Image](https://github.com/user-attachments/assets/4b04e94e-b58a-409b-8a46-214c83015969)
请问代码应该如何改进呢

**Comment by et123456789** [comment]
保存的ano_map是这样的，上面贴错了

![Image](https://github.com/user-attachments/assets/4f5f5da6-3abc-414a-930a-0b5a1db09c6c)

**Comment by et123456789** [comment]
保存的ano_map是这样的，上面贴错了

![Image](https://github.com/user-attachments/assets/67348a97-f567-4df0-a1da-4257d0d5e8a8)

**Comment by et123456789** [comment]
已解决！
